geopandas/array.py

```diff
@@ -1394,6 +1394,8 @@ def __contains__(self, item):
 
 
 def _get_common_crs(arr_seq):
+    # mask out all None arrays (commonly from concat where column is missing)
+    arr_seq = [ga for ga in arr_seq if not ga.isna().any()]
```

Comment:
Should we only mask out all-None arrays _if_ they have no crs set? (because if they have a CRS, then they are not auto-generated by pandas)

Reply:
This makes sense to apply this as narrowly as possible.